py/desispec/scripts/specex.py

```diff
@@ -499,8 +501,8 @@ def mean_psf(inputs, output):
                     val = legval(iu,ytrace[f])
                     ytrace[f] = legfit(ou,val,deg=npar-1)
 
-            hdulist["xtrace"].data = np.median(np.array(xtrace),axis=0)
-            hdulist["ytrace"].data = np.median(np.array(ytrace),axis=0)
+            hdulist["xtrace"].data = np.mean(np.array(xtrace),axis=0)
+            hdulist["ytrace"].data = np.mean(np.array(ytrace),axis=0)
```

Comment:
Efficiency: it appears that `xtrace` and `ytrace` are already arrays due to lines 485-486, in which case this is making another memory copy.  Consider dropping `np.array` or using `np.asarray` instead.

Reply:
done